doc/Attestation.adoc

```diff
@@ -7,32 +7,31 @@ A high level description of the thinking and how this can be used can be found
 at https://developers.yubico.com/PIV/Introduction/PIV_attestation.html
 
 === Usage
-Attestation works through a special key slot called “f9” this comes
-pre-loaded from factory with a key and cert signed by Yubico, but can be
-overwritten.
-After a key has been generated in a normal slot it can be attested by this
-special key, this can be realised by using the yubico-piv-tool action attest:
+The only purposes of PIV attestation is to show that a certain asymmetric key has been generated on device and not imported.
```

Comment:
I'd remove this editorial comment until we resolve that discussion internally.

Reply:
This is taken with slight modification from the PIV attestation documentation linked earlier in this doc here: https://developers.yubico.com/PIV/Introduction/PIV_attestation.html

We can remove this for now, but let's keep a mental note here that we'll need to rectify it with our other documentation for PIV attestation here, as well as the YESDK documentation.

Reply:
Removed - we will iron out additional changes to this page later.